cmd/describe.go

```diff
@@ -0,0 +1,36 @@
+package cmd
+
+import (
+	"github.com/redhat-developer/odo/pkg/component"
+	"github.com/spf13/cobra"
+)
+
+var describeCmd = &cobra.Command{
+	Use:   "describe [component_name]",
+	Short: "Describe the given component",
+	Long:  `Describe the given component.`,
+	Example: ` # Describe nodejs component,
+	ocdev describe nodejs
```

Comment:
two spaces  at the beginning of each line to make it align with the rest of the messages